manual/keycloak-retrieve-token.md

```diff
@@ -0,0 +1,46 @@
+# Keycloak: Retrieve Authorisation Tokens
+
+Based on the following blog post: https://medium.com/@bcarunmail/securing-rest-api-using-keycloak-and-spring-oauth2-6ddf3a1efcc2
+
+## Authorisation Token 
+
+The Keycloak OpenID Connection configuration URL to get details about all security endpoints is: 
+`GET https://{keycloak url}/auth/realms/{realm name}/.well-known/openid-configuration`
```

Comment:
Thanks for adding this. 👍 

Might be worth adding the local one explicitly, since that's the one most people will need regularly: http://localhost:8081/auth/realms/tdr/.well-known/openid-configuration

Reply:
Resolved